components/service/experiments/README.md

```diff
@@ -178,12 +178,14 @@ The experiments records in Kinto contain the following properties:
-| match.locale_country      |        |          | The default locales country.                                                                                                                                                | "USA|DEU"                                                    |
-| match.locale_language     |        |          | The default locales language.                                                                                                                                                | "eng|zho|deu"                                                |
-| match.app_display_version |        |          | The application version.                                                                                                                                                |                                                                |
+| match.app_id              | regex  |          | The app ID (package name)                                                                                                                       | "^org.mozilla.firefox_beta${'$'}|^org.mozilla.firefox${'$'}"   |
+| match.device_manufacturer | regex  |          | The Android device manufacturer.                                                                                                                |                                                                |
+| match.device_model        | regex  |          | The Android device model.                                                                                                                       |                                                                |
+| match.locale_country      |        |          | The default locales country.                                                                                                                    | "USA|DEU"                                                      |
+| match.locale_language     |        |          | The default locales language.                                                                                                                   | "eng|zho|deu"                                                  |
+| match.app_display_version | regex  |          | The application version.                                                                                                                        | `"1.0.0"`                                                      |
+| match.app_min_version     | string |          | The minimum application version, inclusive.                                                                                                     | `"1.0.0"`                                                      |
+| match.app_max_version     | string |          | The maximum application version, inclusive.                                                                                                     | `"1.0.0"`                                                      |
```

Comment:
Nit: It would be nice to use different versions for these, e.g. "1.0.1" & "1.4.2", to highlight how those could be used.